samples/cellular/nrf_cloud_rest_device_message/README.rst

```diff
@@ -112,6 +135,11 @@ This overlay allows the sample and various subsystems that have logging enabled
 Set the :kconfig:option:`CONFIG_NRF_CLOUD_LOG_OUTPUT_LEVEL` option to the log level of messages to send to nRF Cloud, such as ``4`` for debug log messages.
 Set the :kconfig:option:`CONFIG_NRF_CLOUD_REST_DEVICE_MESSAGE_SAMPLE_LOG_LEVEL_DBG` option so that log messages are generated on each button press.
 
+.. _nrf_cloud_rest_device_message_remote_provisioning_overlay:
+
+Remote provisioning overlay
+===========================
+
```

Comment:
this section should include a note about the UUID device id format that is now enabled by using this overlay.